proto/osmosis/incentives/gauge.proto

```diff
@@ -9,48 +9,46 @@ import "osmosis/lockup/lock.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v7/x/incentives/types";
 
-// Gauge is a struct that works as a single unit for each distribution of
-// incentives defined by x/incentives module.
+// A Gauge is an object that stores and distributes yields to recipients who satisfy certain conditions.
+// Currently gauges support conditions around the duration for which a given denom is locked.
 message Gauge {
-  // ID is the unique id of the gauge.
-  // The ID of the gauge is decided upon gauge creation, incrementing by 1 for
-  // every gauge.
+  // Unique ID of a Gauge
   uint64 id = 1;
-  // IsPerpetual defines whether the gauge is perpetual or not.
-  // A non-perpetual gauge distributes the incentive tokens equally per epoch
-  // while the gauge is in an active period.
-  // A perpetual gauge distributes all tokens at a single distribution, mainly
-  // used to distribute minted osmo to lp token stakers.
+  // Flag to show if it's a perpetual or non-perpetual gauge
```

Comment:
I thought the convention was to prefix according to the name of the field like `is_perpetual`?

Example in epochs: https://github.com/osmosis-labs/osmosis/blob/a364494a3ac7f9d369b88ac0c3c29cb0f3542a16/proto/osmosis/epochs/genesis.proto#L10-L18

Reply:
If this is the case I will go back and add the prefixes!

Reply:
I went back and did this for all the functions since Dev told me that was golang convention, I didn't realize that was also the protobuf convention (you are probably right)

Reply:
Would be cool if we could add this in the linter too :)

Reply:
What protobuf does is it generates go code from `proto` files, including comments. For example, for the epochs proto code I linked, the Go would be:
https://github.com/osmosis-labs/osmosis/blob/a364494a3ac7f9d369b88ac0c3c29cb0f3542a16/x/epochs/types/genesis.pb.go#L30-L34
- note that comments are ported over as well

Since we would like to maintain the Go standards, we also want to have the comments in proto files be in the same way as Go so that `pb.go` files are generated with the right style.

The only problem with proto -> Go comment conversion is that it does not cover the case according to Go conventions.
For example, `identifier` should be converted to `Identifier` in Go. However, it preserves the proto case. We might want to address this in the future but this is not a big deal at all for now.